src/include/bitstring.i

```diff
@@ -231,18 +231,6 @@ __bit_getv(uint8_t *bitf, uint64_t entry, uint8_t width)
 	uint8_t value;
 	uint64_t bit;
 
```

Comment:
I would add a comment somewhere in this file so that someone knows later why this was done.  I could see someone trying to undo this a few years from now.